test/unit/health_check/check_file_parser_test.rb

```diff
@@ -8,78 +8,54 @@ def checks(data)
     end
 
     should "read the supplied file and return a list of checks" do
-      data = "Monthly searches,When I search for...,Then I...,see...,in the top ... results
-600,a,should,https://www.gov.uk/a,1
-500,b,should,https://www.gov.uk/b,1
-"
-      expected = [SearchCheck.new("a", "should", "/a", 1, 600), SearchCheck.new("b", "should", "/b", 1, 500)]
-      assert_equal expected, checks(data)
-    end
+      data = <<~END
+        Tags,When I search for...,Then I...,see...,in the top ... results,Current position,Link,Last reviewed (Ctrl ;),Word count,Source,Duplicates?
+        test,a,should,https://www.gov.uk/a,1
+        test,b,should,https://www.gov.uk/b,1
+      END
 
-    should "skip rows that don't have an integer for the top N number" do
-      data = "Monthly searches,When I search for...,Then I...,see...,in the top ... results
-500,b,should,https://www.gov.uk/b,mistake
-"
-      expected = []
+      expected = [SearchCheck.new("a", "should", "/a", 1, 1, %w(test)), SearchCheck.new("b", "should", "/b", 1, 1, %w(test))]
       assert_equal expected, checks(data)
     end
 
-    should "accept Monthly searches values containing commas" do
-      data = "Monthly searches,When I search for...,Then I...,see...,in the top ... results
-\"6,000\",a,should,https://www.gov.uk/a,1
-"
-      expected = [SearchCheck.new("a", "should", "/a", 1, 6_000)]
-      assert_equal expected, checks(data)
-    end
+    should "skip rows that don't have an integer for the top N number" do
```

Comment:
Yeah it's not something I've changed.

I think it's logged here: https://github.com/alphagov/rummager/blob/aa4e73f88ef893d379361e0fd3096d793b977252/lib/health_check/check_file_parser.rb#L30